inductiva/utils/format_utils.py

```diff
@@ -28,6 +38,26 @@ def bytes_formatter(n_bytes: int) -> str:
     return f"{bytes_formatter:.2f} PB"
 
 
+def _supports_ansi():
+    if sys.platform.startswith("win"):
+        return "TERM" in os.environ and os.environ["TERM"] == "xterm"
+    return hasattr(sys.stdout, "isatty") and sys.stdout.isatty()
+
+
+def emphasis_formater(string_to_emphasize, emphasis):
```

Comment:
What other types of emphasis are supported? The name points to general ones, the body points to colors.

Reply:
At the moment just the ones we define

Reply:
So maybe the naming should be more explicit.